README.md

```diff
@@ -6,7 +6,7 @@ Demo coming
 
 ## Installation
 
-Run `npm i react-scroll-to-top`
+Run `npm i react-scroll-to-top --save`
```

Comment:
I'm pretty sure NPM saves packages by default now, don't think this is needed anymore